pkg/ebpf/manager.go

```diff
@@ -45,6 +41,14 @@ func NewManager(mgr *manager.Manager, modifiers ...Modifier) *Manager {
 	}
 }
 
+// NewManagerWithDefault creates a manager wrapper with default modifiers.
+func NewManagerWithDefault(mgr *manager.Manager, modifiers ...Modifier) *Manager {
```

Comment:
I still think this function is unnecessary, and would prefer to keep the previous behavior on `NewManager` to use defaults if no modifiers are passed.

Reply:
I think this function has a higher chance of accidentally using duplicate modifiers. If the default set changes from A to A+B, and someone was using `NewManagerWithDefault(mgr, B)`, then they now get a duplicate of B unless we also check/modify all callsites.